app/ide-desktop/lib/file-watcher/src/index.ts

```diff
@@ -0,0 +1,31 @@
+import yargs from 'yargs'
+import { hideBin } from 'yargs/helpers'
+import chokidar from 'chokidar'
+
+const args = yargs(hideBin(process.argv))
+    .option('path', {
+        alias: 'p',
+        type: 'string',
+        default: '.',
+        description: 'File path to watch',
+    })
+    .option('verbose', {
+        alias: 'v',
+        type: 'boolean',
+        default: false,
+        description:
+            'If false, only reports add, change, and unlink events.If true, reports all events.',
+    })
+    .parseSync()
+
+if (args.verbose) {
+    chokidar.watch(args.path).on('all', (event, path) => {
+        console.log({ event, path })
+    })
+} else {
+    chokidar
+        .watch(args.path)
+        .on('add', path => console.log({ event: 'add', path }))
+        .on('change', path => console.log({ event: 'change', path }))
+        .on('unlink', path => console.log({ event: 'unlink', path }))
```

Comment:
I understand that this is not finished, or is it? There should be docs describing what is the idea behind that and that the consumer should consume things from STD OUT.

Reply:
I believe this is complete or which events are we missing? Added some documentation!